### ISSUE TITLE
CeeSection glitches

### ISSUE THREAD

**Issue by Agent6-6-6** [robbievanleeuwen/section-properties#12]
**Describe the bug**
Hi Robbie, perhaps this is a couple of unintended use cases, but I notices a few glitchy things occuring with the CeeSection class:-

Case 1
If you use a defined CeeSection with zero outside radius (r=0, n_r=1), i.e. for what I presumed would be a square outside corner (similar to defining an I section with no root radii), the code exits without doing anything or raising any exceptions. 

Sections like an I section you can set the root radii to 0, with n_r=1 for a square internal corner. Might be something to note in the documentation that r_out>0 for CeeSection. 

Or alternatively alter code to pickup this special use case, extruded sections might have square corners for example. You can work around it by specifying sufficiently small radii, but then sometimes you run into an exception related to precision errors because presumably the mesh is simply too fine with a very small radii to model a square corner. 

This may also apply to some other section types such as RHS with external/exterior corners, but I have not specifically checked.

Case 2
Also there should possibly be a check on 'l' the lip length being greater than the radii, otherwise you get some weirdness. perhaps an option if L = 0 then the lip and the corner radius are not added, or raise an exception when between 0 and the outside radii to avoid the issues illustrated below? 

![Figure_2](https://user-images.githubusercontent.com/36808164/60579132-97fccc80-9dd6-11e9-88bf-04783228e15d.png)

**To Reproduce**
Run this to show code exiting without any exception or feedback at all
```
import sectionproperties.pre.sections as sections
from sectionproperties.analysis.cross_section import CrossSection
from sectionproperties.pre.pre import Material

t = 1.16
d = 150
b = 40
l = 10
r_out = 0
n_r = 1
channel1 = sections.CeeSection(d=d, b=b, l=l, t=t, r_out=r_out, n_r=n_r)

# single channel
geometry = channel1
geometry.clean_geometry(verbose=True)

mesh = geometry.create_mesh(mesh_sizes=[0.25])

geometry.clean_geometry(verbose=True)

section = CrossSection(geometry, mesh)
section.plot_mesh()
section.display_mesh_info()
section.calculate_geometric_properties(time_info=True)
section.calculate_plastic_properties(time_info=True)
section.calculate_warping_properties(time_info=True)

section.display_results(fmt='.3f')
```
Run this to show the weird corner issues when 'l' is smaller than the outside radii.

```
t = 1.16
gap = 0.1
d = 150
b = 40
l = 1
r_out = 1
n_r = 4 
channel1 = sections.CeeSection(d=d, b=b, l=l, t=t, r_out=r_out, n_r=n_r)

# single channel
geometry = channel1
geometry.clean_geometry(verbose=True)

mesh = geometry.create_mesh(mesh_sizes=[0.25])

geometry.clean_geometry(verbose=True)

section = CrossSection(geometry, mesh)
section.plot_mesh()
section.display_mesh_info()
section.calculate_geometric_properties(time_info=True)
section.calculate_plastic_properties(time_info=True)
section.calculate_warping_properties(time_info=True)

section.display_results(fmt='.3f')
```

Thanks!

**Comment by robbievanleeuwen** [comment]
Hi Agent,

Thanks for bringing these to my attention - I've just gone through all the other sections and made sure that they work for zero radii (also fixed the RHS when r_out < t). I'll have a look at the Cee and Zed sections shortly.

Robbie

**Comment by robbievanleeuwen** [comment]
Case 1 fixed with 10ab90e. Also captured case where r_out < t - internal radius is set to zero.

**Comment by robbievanleeuwen** [comment]
Case 2 fixed with 3b1698c.

If the lip length is less than the outer radius an exception is raised.

If the lip length equals the outer radius, the lip is ignored.
